README.md

```diff
@@ -8,6 +8,11 @@
   <br/><br/>
 </p>
 
+![Testing Status](https://github.com/withastro/astro/actions/workflows/test-hosts.yml/badge.svg)
```

Comment:
I don't think we're currently tracking our hosted test suite as a success metric, so I might remove this from the README.

```suggestion
```